homeassistant/components/amberelectric/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/amberelectric",
   "iot_class": "cloud_polling",
   "loggers": ["amberelectric"],
-  "requirements": ["amberelectric==1.1.1"]
+  "requirements": ["amberelectric==2.0.9"]
```

Comment:
Version 2.0.9 on PyPI still has a `setup.py` file in the source distribution. If I run `poetry build` in the amberelectric.py project root there's no `setup.py` included in the generated source distribution. I think your build and publish procedure may need to be reviewed.

Reply:
Yep - looks like my build env was hosed.

Checked it out on a different machine, and it built properly. 🤷‍♀️